src/include/snapper/dialogs.rb

```diff
@@ -25,12 +25,9 @@
 # Authors:	Jiri Suchomel <jsuchome@suse.cz>
 
 module Yast
-
```

Comment:
Who the hell is bbatsov. We use https://github.com/SUSE/style-guides/blob/master/Ruby.md.